upgrade.md

```diff
@@ -79,6 +79,22 @@ The constructor signature of the `Illuminate\Auth\Access\Response` class has cha
 
 The `Illuminate\Contracts\Auth\Access\Gate` contract has received a new `inspect` method. If you are implementing this interface manually, you should add this method to your implementation.
 
+#### The `Illuminate\Auth\Access\HandlesAuthorization` Trait
+
+**Likelihood Of Impact: Low**
+
+The `Illuminate\Auth\Access\HandlesAuthorization` trait's method `deny()` has been update to return `\Illuminate\Auth\Access\Response`. Previously it used to throw an exception. 
```

Comment:
1. `has been update` should be `has been updated` or `has been changed`
2. `Previously it used to` might read better: `Previously it would`

Reply:
@drbyte 
Thanks, pushed new changes